### PR TITLE
feat: use open-web-calendar to view the W3C calendar

### PR DESCRIPTION
**Pull request by danielpeintner**
Uses https://github.com/niccokunzmann/open-web-calendar to render the calendar at https://www.w3.org/groups/wg/wot/calendar/export

Some things that do not work as expected.
* (Zoom) Links etc are not working as expected
* week view starts at 0 o clock ... which is most of the time wrong --> I think we should use the month view

Anyhow,  I think it is way better as what we do now!

fixes https://github.com/w3c/wot-marketing/issues/268

### CODE REVIEW

**Comment by danielpeintner** [comment]
see https://deploy-preview-495--wot-marketing.netlify.app/activities/meetings/ for the calendar

**Comment by egekorkan** [comment]
Looks really good already! Regarding zoom, I see Zoom as the location so we should avoid that in general I guess.

**Comment by danielpeintner** [comment]
> You can use `starting_hour` to start at better times as seen [here](https://open-web-calendar.quelltext.eu/templates/#mobile-one-day-calendar-feed)

I looked into `starting_hour` but this works just fine with a given timezone.

I [pushed an update](https://github.com/w3c/wot-marketing/pull/495/commits/488bb0e0e5e66ffb2281b3b31b0fa775e1e307d4) to show what I mean.
https://deploy-preview-495--wot-marketing.netlify.app/activities/meetings/

It works fine for Europe but it cuts off entries for other timezones (e.g. try America).
This means we would need to adjust the `starting_hour` depending on each each timezone. I think this is brittle and not worth it... but I am open to other opinions.

**Comment by lu-zero** [comment]
We may ask upstream about that and ask if `starting_hours` can take always an UTC time.

**Comment by lu-zero** [comment]
Opened https://github.com/niccokunzmann/open-web-calendar/issues/383

File: docs/activities/meetings/index.html [hunk 1]
```diff
@@ -5,69 +5,25 @@
...
+  // * target=_blank // Links
+  var newSrc = "https://open-web-calendar.hosted.quelltext.eu/calendar.html?url=https://www.w3.org/groups/wg/wot/calendar/export&start_of_week=work";
   
-  // set the selected one visible
-  if (document.getElementById(selVal)) {
-    document.getElementById(selVal).style.display = '';
-  } else {
-    document.getElementById("UTC").style.display = ''; // default
+  // XXX should be removed
+  newSrc += "&tab=week&starting_hour=11"; // to make it work properly we need to adjust the hours based on the timezone
```
Comment (egekorkan) [inline docs/activities/meetings/index.html:20]: While the upstream issue is being discussed, can't we use the previous timezone finding logic to enter the starting hours dynamically?

Comment (egekorkan) [reply docs/activities/meetings/index.html:20]: That is also true and makes everything much simpler :D

Comment (lu-zero) [reply docs/activities/meetings/index.html]: It does look pretty indeed.

**Comment by danielpeintner** [comment]
Shall we approve and merge this PR asynchronously?
We did not have any calls for a while and we will not have one this week..

I think the PR improves the current situation a lot (updates entries automatically etc) and reduces our workload also..

**Comment by egekorkan** [comment]
> and we will not have one this week..

There will be one today :)

**Comment by danielpeintner** [comment]
> There will be one today :)

👍

Mhh, it gets confusing since we say "Wednesday-Friday, June 12-14 - Kaz will be travelling for Interop 2024 event" 
[here](https://www.w3.org/WoT/IG/wiki/Main_WoT_WebConf#Cancellations_and_Schedule_Updates)

**Comment by egekorkan** [comment]
Given that no one has answered to https://lists.w3.org/Archives/Member/member-wot-wg/2024May/0003.html to indicate their usage preference of the old view, this change should not affect anyone. Thus, we have added the minor label and are going to merge this PR.

**Pull request merged**